main.css

```diff
@@ -0,0 +1,97 @@
+.heading {
+    font-family: 'Apple-Chancery', cursive;
```

Comment:
Thank you so much for all your help. I am not very sure how to put all google fonts.. Please help me.. Thanks..